tests/e2e/online_tests.py

```diff
@@ -210,7 +246,7 @@ def test_users_add_to_group(self):
         command = "addusers"
         filename = os.path.join("tests", "assets", "usernames.csv")
         arguments = [command, groupname, "--users", filename]
-        arguments.append("--continue-if-exists")
+        # classic doesn't have this arg arguments.append("--continue-if-exists")
```

Comment:
I would say either delete this line or put it behind an if-statement, not a comment. (Same for other places it occurs).